packages/typespec-vscode/ThirdPartyNotices.txt

```diff
@@ -9,9 +9,15 @@ components are set forth below. Microsoft reserves all rights not expressly
 granted herein, whether by implication, estoppel or otherwise.
 
 1. balanced-match version 1.0.2 (https://github.com/juliangruber/balanced-match)
-2. brace-expansion version 2.0.1 (https://github.com/juliangruber/brace-expansion)
-3. minimatch version 5.1.6 (https://github.com/isaacs/minimatch)
-4. semver version 7.6.2 (https://github.com/npm/node-semver)
+2. balanced-match version 1.0.2 (https://github.com/juliangruber/balanced-match)
+3. brace-expansion version 2.0.1 (https://github.com/juliangruber/brace-expansion)
```

Comment:
feels unexpected change here

Reply:
this is just my normal TPN diff that always exists for reasons I have never bothered to diagnose. I've undone this change.